domains/transfer/Transfer.API/API/Options/ConnectionInvitationCleanupServiceOptions.cs

```diff
@@ -7,6 +7,6 @@ public class ConnectionInvitationCleanupServiceOptions
 {
     public const string Prefix = "ConnectionInvitationCleanupService";
 
-    [Required]
+    [Range(typeof(TimeSpan), "00:00:02", "02:00:00")]
```

Comment:
Why this? The required flag forces you to set it

Reply:
The required flag does nothing, to enforce it during runtime. It is omitted, as the code enforcing the presence of this variable exists in program.cs